src/dom/element.ts

```diff
@@ -220,15 +267,21 @@ export class Element extends Node {
   }
 
   setAttribute(name: string, value: any) {
-    this.attributes[name] = "" + value;
+    const strValue = String(value);
```

Comment:
Why are you using the `String()` function instead of `"" + value` here?

Reply:
I find the `String()`, `Number()`, etc. functions more clear than forcing an implicit typecast. But I'll change it back if you prefer.

Reply:
Yeah, change it to the coercion thanks